opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 The OpenTracing Authors
+ * Copyright 2016-2017 The OpenTracing Authors
```

Comment:
Making license plugin happy.

(PR editing `MockSpan` was created in 2016 and merged in 2017).